packages/editor/src/browser/editor-decorations-service.ts

```diff
@@ -155,3 +156,77 @@ export interface DecorationOverviewRulerOptions {
      */
     position?: OverviewRulerLane;
 }
+
+export class DecorationStyle implements Disposable {
```

Comment:
it is still generic, there are tree and editor decorations for example

Reply:
thanks! I also changed name of NS.